backend/WebApi/Auth/JwtService.cs

```diff
@@ -98,7 +101,29 @@ public static TokenValidationParameters TokenValidationParameters(JwtOptions jwt
 
             ValidateIssuer = true,
             ValidateAudience = true,
-            ValidateLifetime = true
+            ValidateLifetime = true,
+        };
+    }
+
+    public static ILfWebContext BuildUserContext(ClaimsPrincipal user)
```

Comment:
this should probably only be responsible for returning the `LfUser` instead of the whole Context because this function is in the `JwtService`

Reply:
Totally 👍